src/main/java/com/sparta/spangeats/domain/review/controller/ReviewController.java

```diff
@@ -37,7 +37,7 @@ public ResponseEntity<List<ReviewResponse>> getALlForStore(@PathVariable Long st
     }
 
     //리뷰 전체 조회(회원별) - 날짜순, 디폴트: 최신
-    @GetMapping("/get/review/member")
```

Comment:
이미 http method 자체로 `get`해온다는 의미가 있기 때문에 url에 `get`을 넣을 필요없을 거 같습니다!
RESTful한 api 설계는 최대한 url에 동사를 넣는 것을 지양하기 위해 '리소스` 위주로만 url에 넣고 http method로 그 행동을 표현하려고 하는 편이에요. 
```suggestion
    @GetMapping("/review/member")
```